linux/snap/snapcraft.yaml.in

```diff
@@ -24,9 +24,9 @@ parts:
       COQREGTESTING=y OPAMROOT=$SNAPCRAFT_PART_INSTALL/coq-platform/ ./coq_platform_make.sh @@PLATFORM_ARGS@@ -dumplogs
       # cleanup
       OPAMROOT=$SNAPCRAFT_PART_INSTALL/coq-platform/ opam clean
-      find $SNAPCRAFT_PART_INSTALL/coq-platform/@@PLATFORM_RELEASE@@/ \( -name '*.byte.exe' -o -name '*.byte' -o -name '*.cm[aioxt]' -o -name '*.cmxa' -o -name '*.[oa]' -o -name '*.cmti' -o -name '*.glob' \) -type f  -delete
-      find $SNAPCRAFT_PART_INSTALL/coq-platform/@@PLATFORM_RELEASE@@/bin/ -maxdepth 1 -mindepth 1 \! \( -name 'coq*' -o -name 'clight*' -o -name 'gappa' -o -name 'eprover' -o -name 'z3_tptp' \) -exec rm -f {} \;
-      find $SNAPCRAFT_PART_INSTALL/coq-platform/@@PLATFORM_RELEASE@@/lib/ -maxdepth 1 -mindepth 1 \! \( -name 'coq*' -o -name 'stublibs' \) -exec rm -rf {} \;
+      find $SNAPCRAFT_PART_INSTALL/coq-platform/@@PLATFORM_RELEASE@@/ \( -name '*.byte.exe' -o -name '*.byte' -o -name '*.cm[aioxt]' -o -name '*.cmxa' -o -name '*.[oa]' -o -name '*.cmti' -o -name '*.glob' \) -type f -delete
+      find $SNAPCRAFT_PART_INSTALL/coq-platform/@@PLATFORM_RELEASE@@/bin/ \( -name 'ocaml*' -o -name 'camlp5*' -o -name 'mkcamlp5' -o -name 'ocpp5' -o -name 'cppo' -o -name 'safe_camlp4' -o -name 'ppxfind' -o -name 'gdk_pixbuf_mlsource3' -o -name 'lablgladecc3' \) -type f -delete
+      find $SNAPCRAFT_PART_INSTALL/coq-platform/@@PLATFORM_RELEASE@@/lib/ -maxdepth 1 -mindepth 1 \( -name 'coq' -o -name 'stublibs' \) -type d -exec rm -rf {} \;
```

Comment:
Didn't you forget the negation in this lase one?

Reply:
I mean, you must keep `coq-core` and `stublibs` for example

Reply:
Umm yes - I forgot a negation in here ... I first wanted to do a negative list for lib as well, but then found that here positive makes more sense.